src/python/pants/backend/codegen/protobuf/scala/scalapbc.py

```diff
@@ -31,12 +52,30 @@ def register_options(cls, register):
                 "A list of addresses to `jvm_artifact` targets for the runtime "
                 "dependencies needed for generated Scala code to work. For example, "
                 "`['3rdparty/jvm:scalapb-runtime']`. These dependencies will "
-                "be automatically added to every `protobuf_sources` target. At the very leasst, "
+                "be automatically added to every `protobuf_sources` target. At the very least, "
                 "this option must be set to a `jvm_artifact` for the "
                 f"`com.thesamet.scalapb:scalapb-runtime_SCALAVER:{cls.default_version}` runtime library."
             ),
         )
+        register(
+            "--jvm-plugin-artifacts",
+            type=list,
+            member_type=str,
+            # TODO: Add example of how to invoke a plugin like `fs2-grpc`.
```

Comment:
Could probably do this inline with an embedded code block.

Reply:
Done.